resources/keywords/deploy_instance_dev.resource

```diff
@@ -3,15 +3,16 @@ Resource    ../keywords/create_provider_account.resource
 Resource    ../object_repo/developer_catalog_obj.resource
 
 *** Variables ***
-${newProject}      ${EMPTY}
+${newProject}        ${EMPTY}
 ${instanceName}      ${EMPTY}
 
 *** Keywords ***
 User Selects A Project For Database Connection
     Run Keyword If    '${newProject}'=='${EMPTY}'
     ...    Create New Project
-    ...  ELSE
+    ...  ELSE    Run Keywords
     ...    Log To Console    Project ${newProject} exists
```

Comment:
My bad, we can remove the "Log To Console" keyword

Reply:
Can we keep this around for a little while, just until we cover all the corner cases that we might encounter.
This helps a bit in debugging and we can remove this in subsequent PRs, will that be okay?

Reply:
Sure